python_modules/dagster/dagster/_core/execution/submit_asset_runs.py

```diff
@@ -268,7 +277,9 @@ def submit_asset_runs_in_chunks(
     asset_graph: ExternalAssetGraph,
     debug_crash_flags: SingleInstigatorDebugCrashFlags,
     logger: logging.Logger,
-) -> Iterator[Optional[Sequence[Tuple[RunRequest, DagsterRun]]]]:
+    backfill_id: Optional[str] = None,
+    catch_code_location_load_errors: bool = False,
```

Comment:
i only see one callsite of this method - is there a reason it needs to be toggleable?

Reply:
Yeah only one callsite, i'm ok with making it non-toggleable